index.html

```diff
@@ -0,0 +1,399 @@
+                                        2
+                                    </option>
+                                    <option>
+                                        3
+                                    </option>
+                                </select>
+                            </span>
+                            <span>
+                                <select class="dropdown_select overseas_select">
+                                    <option>
+                                        1
+                                    </option>
+                                    <option>
+                                        2
+                                    </option>
+                                    <option>
+                                        3
+                                    </option>
+                                </select>
+                            </span>
+                        </div>
+                        <div class="">
+                            <textarea cols="80" name="name" rows="4" class="details_oversea_textarea">
+                            </textarea>
+                        </div>
+                    </div>
+                    <div class="submit_button">
+                        <button name="button" type="submit">
+                            <b>
+                                SUBMIT
+                            </b>
+                        </button>
+                    </div>
+                </form>
+
+            </section>
+    
+        </main>
+    <footer>
+    <div id="footer">
+        <div id="footer_left">
+            <img alt="" src="assets/images/footer_left.png"/>
+        </div>
+        <div id="footer_middle">
+            <ul>
+                <li>
+                    <a class="smoothScroll" href="#content">
+                        Home
+                    </a>
+                </li>
+                <li>
+                    <a class="smoothScroll" href="#content_about_us">
+                        About Us
+                    </a>
+                </li>
+                <li>
+                    <a class="smoothScroll" href="#content_about_the_founder">
+                        Founder
+                    </a>
+                </li>
+                <li>
+                    <a class="smoothScroll" href="#content_business_logic">
+                        Business Logic
+                    </a>
+                </li>
+                <li>
+                    <a class="smoothScroll" href="#content_official_launch">
+                        When
+                    </a>
+                </li>
+                <li>
+                    <a class="smoothScroll" href="#content_write_to_us">
+                        Write to Us
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div id="footer_right">
+            <img alt="" src="assets/images/footer_right.png"/>
+        </div>
+    </div>
+</footer>
```

Comment:
Footer breaks on window resize. Please correct it. 
Would be better if the pattern on left and right can be included as background images.